sites/fast-website/src/app/css/style.css

```diff
@@ -148,6 +148,10 @@ li {
   background: transparent;
 }
 
+.logo:focus::part(content)::before {
```

Comment:
@khamudom can you change this to use `${focusVisible}`? We can address the others after, but I'd love to at least have it in for the logo

Reply:
update to use focus-visible for now, because this is in the styles.css file